src/cascade/input_data/emr.py

```diff
@@ -0,0 +1,219 @@
+from textwrap import dedent
+
+from scipy.interpolate import InterpolatedUnivariateSpline, LSQBivariateSpline
+
+import numpy as np
+import pandas as pd
+
+from cascade.input_data.db.mortality import get_cause_specific_mortality_data
+from cascade.input_data.db.demographics import age_groups_to_ranges, get_mean_years
+from cascade.stats import meas_bounds_to_stdev
+
+from cascade.core import getLoggers
+
+CODELOG, MATHLOG = getLoggers(__name__)
+
+
+def _prepare_prevalence(observations):
+    prevalence = observations.query("measure == 'prevalence' and sex_id in [1, 2, 3]")
+
+    zero_means = prevalence["mean"] == 0
+    if np.any(zero_means):
+        not_quite_zero = 1e-6
```

Comment:
If I were to do this, would this be ok, or would I be asked to create a module constant?  

sometimes, looks like we use 1e-9, here 1e-6.

Reply:
I don't think I would ask you to do that, @adolgert might? I think the exact value here isn't important and is unlikely to change or be tuned.